workflows/cellranger/cellranger_count.wdl

```diff
@@ -176,7 +176,7 @@ task run_cellranger_count {
                 fout.write('fastqs,sample,library_type\n')
                 for i, directory in enumerate('~{input_fastqs_directories}'.split(',')):
                     directory = re.sub('/+$', '', directory) # remove trailing slashes
-                    call_args = ['strato', 'cp', '--backend', '~{backend}', '-m', '-r', directory + '/' + samples[i], '.']
+                    call_args = ['strato', 'cp', '--backend', '~{backend}', '-m', '-r', directory + '/' + samples[i] + '/', './samples[i]/']
```

Comment:
This won't work for GCP. Haven't tested for local backend.

Since this is copying a directory. So using `strato sync` command would hopefully simplify things. Please use
```
call_args = ['strato', 'sync', '--backend', '~{backend}', '-m', directory + '/' + samples[i], '.' + samples[i]]
```
(notice that you have a typo: `'./samples[i]'` should be `'.' + samples[i]`)